cmd/kube-controller-manager/app/options/options.go

```diff
@@ -442,6 +432,5 @@ func createRecorder(kubeClient clientset.Interface, userAgent string) record.Eve
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(glog.Infof)
 	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
-	// TODO: remove dependency on the legacyscheme
-	return eventBroadcaster.NewRecorder(legacyscheme.Scheme, v1.EventSource{Component: userAgent})
+	return eventBroadcaster.NewRecorder(kubectrlmgrconfigscheme.Scheme, v1.EventSource{Component: userAgent})
```

Comment:
This has nothing to do with component configs. Would prefer to use a local scheme here. Which types do we need in that scheme? We need corev1, don't we?

Reply:
Yes. Do you means we just need `scheme` for `corev1`?

Reply:
fixed